packages/form-js-editor/src/render/components/editor-form-fields/EditorText.js

```diff
@@ -0,0 +1,25 @@
+import { Text, isExpression } from '@bpmn-io/form-js-viewer';
+
+import { iconsByType } from '../icons';
+
+export default function EditorText(props) {
+
+  const text = props.field.text || '';
+
+  const Icon = iconsByType['text'];
+
+  if (!text) {
+    return <div class="fjs-form-field-placeholder"><Icon viewBox="0 0 54 54" />Text view is empty</div>;
+  }
+
+  if (isExpression(text)) {
+    return <div class="fjs-form-field-placeholder"><Icon viewBox="0 0 54 54" />Text view is populated by an expression</div>;
+  }
+
+  return <Text { ...props } />;
+}
+
+// TODO: How could we get rid of the redundance here this ?
```

Comment:
Using `classes` maybe? But that would increase the complexity slightly, so I'd ok with having the redundancy here.

Reply:
One thing I've considered also is to group all these meta-properties into a single object. But maybe a refactoring for the future :)